### ISSUE TITLE
get_specfile_macros: Assertion failed

### ISSUE THREAD

**Issue by ukulekek** [rpminspect/rpminspect#158]
Hello
And some problem with specfile
```
$ rpminspect -v systemd-219-73.el7_8.6 systemd-219-73.el7_8.9
...
Running license inspection...        pass
Running emptyrpm inspection...       pass
Running lostpayload inspection...    pass
Running metadata inspection...       pass
Running manpage inspection...        pass
Running xml inspection...            pass
Running elf inspection...            pass
Running desktop inspection...        pass
rpminspect: ../lib/macros.c:99: get_specfile_macros: Assertion `(strcmp(entry->data, SPEC_MACRO_DEFINE) == 0 || strcmp(entry->data, SPEC_MACRO_GLOBAL) == 0)' failed.
```
* rpminspect-1.0-0.1.202006161650git.fc31.x86_64
* librpminspect-1.0-0.1.202006161650git.fc31.x86_64
* rpminspect-data-redhat-1.0-0.1.202005291509git.fc31.noarch

**Comment by ukulekek** [comment]
Another reproducer:
```
$ rpminspect -v cri-o-1.18.1-11.dev.rhaos4.5.git4b59581.el8
```

**Comment by dcantrell** [comment]
Both of these are also fixed by commit 9d506f5c683a21168d99c248237d59c2555757db regarding the disttag inspection.
